backend/src/validation-schemas/sign-up/sign-up.validation-schema.ts

```diff
@@ -0,0 +1,12 @@
+import * as Yup from 'yup';
+
+const signUpSchema = Yup.object().shape({
+  name: Yup.string().required('Full name is required'),
```

Comment:
Also use max validation rule

For example 100 would be enough

Reply:
Thanks for the tip. Added new rule.